internal/bundler/bundler.go

```diff
@@ -81,6 +81,7 @@ type parseArgs struct {
 	results         chan parseResult
 	inject          chan config.InjectedFile
 	skipResolve     bool
+	extras          config.ExtraConfig
```

Comment:
"extras" isn't great naming :/ I have plans for future properies, so I'm trying to keep it all in a general spot.